extensions/amp-story-interactive/0.1/amp-story-interactive-poll.js

```diff
@@ -79,6 +81,13 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
     return this.rootEl_;
   }
 
+  /** @override */
+  layoutCallback() {
+    return this.adaptFontSize_(dev().assertElement(this.rootEl_)).then(() =>
```

Comment:
So, as far as I understand it, this will not adapt as the window is resized, correct?

How does that feel?

Reply:
I didn't make it recalculate on layout changes because the styles are in ems, so if the component keeps the same width to height ratio, then options that have 2 lines will still have 2 lines (and the font will keep the correct size). The only way that recalculating the font-size would make sense is if all this conditions meet:
1) The font-size is overriden from the default value
2) The width is not specified in ems, but is also not specified in a unit that scales with the font-size (eg: px vs %)
3) The user resizes the screen _while_ viewing the story, which doesn't happen much for non-developers (?)

Do you think we should recalculate in these scenarios? Binary polls don't recalculate currently (but could add it).

Reply:
Makes sense, I was just wondering how bad it was in the (3) case.  If it's horrible, maybe we can adjust, but otherwise I agree it's not a core use case.

Reply:
Ok in the very worst case scenario it would look like the first screenshot where the option has 2 lines but it keeps the large font-size. I don't think it's that bad anyways, given that it won't show like that in any story that meets all 3 conditions *at the same time*. I'll say doing it in the layoutCallback is the most efficient option and in most cases conditions 1 and 2 won't show up, so less code that has to be ran when resizing.